libs/application/templates/financial-aid/src/fields/Summary/SummaryForm.tsx

```diff
@@ -52,6 +56,19 @@ const SummaryForm = ({ application, goToScreen }: FAFieldBaseProps) => {
     }
   }, [externalData.nationalRegistry?.data?.municipality])
 
+  const { sendSpouseEmail } = useEmail(application)
+
+  if (hasSpouse(answers, externalData)) {
+    setBeforeSubmitCallback &&
+      setBeforeSubmitCallback(async () => {
+        const response = await sendSpouseEmail()
+        if (response) {
+          return [true, null]
+        }
+        return [false, 'Failed to send emails']
+      })
+  }
+
```

Comment:
Just an overall question why are we not handling sendSpouseEmail just in the backend? When creating a application, check if applicant has spouse and send email there from, why here in the application system?

Reply:
We need to do it like this because the application is not sent to our backend until both the applicant and spouse have submitted. When an applicant (that has spouse) submits it is only saved in the AS database, but we need to send email to the spouse when this happens to notify the spouse that the application is waiting.

Reply:
Ahh of course I see, however are we sure we want to not allow applicant to send application if the email fails?

Reply:
Maybe just have a state, and if it fails to give the opportunity to send again or send a link himself or some?